tests/testthat/test-clean_redcap_long.R

```diff
@@ -309,7 +309,7 @@ test_that("convert_mixed_instrument works", {
   expected_out <- tibble::tribble(
     ~record_id, ~redcap_repeat_instrument, ~redcap_repeat_instance, ~mixed_structure_variable,
     ~repeat_form_variable, ~mixed_repeat_var,
-    1, "mixed_structure_form", 1, "A", NA, NA,
+    1, "mixed_structure_form", NA, "A", NA, NA,
```

Comment:
See comment above about no longer using single instance for non repeat treatment.

Reply:
Not applicable, ignore.